tests/test_travis_mf_multiplePS/ps_test_mf.cpp

```diff
@@ -0,0 +1,11 @@
+#include <Configuration.h>
+#include <Tasks.h>
+
+cirrus::Configuration config = cirrus::Configuration("configs/jester.cfg");
+int main() {
+  cirrus::PSSparseServerTask st(
+      (1 << config.get_model_bits()) + 1, config.get_minibatch_size(),
+      config.get_minibatch_size(), config.get_num_features(), 2, 1, "127.0.0.1",
+      1338);
```

Comment:
If the only thing that changes is the port number, make this a command line arg and just use one file (i.e., don't see the need for ps_test_mf2)

Reply:
Fixed.